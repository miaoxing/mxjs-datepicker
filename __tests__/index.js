import React from 'react';
import {render} from '@testing-library/react';
import DatePicker from '..';

describe('DatePicker', () => {
  test('basic', () => {
    const result = render(<DatePicker/>);
    expect(result.container).toMatchSnapshot();
  });
});

