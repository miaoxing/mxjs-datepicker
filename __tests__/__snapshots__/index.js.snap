// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker basic 1`] = `
<div>
  <div
    class="css-1fnqa59-StyledDatePicker e16eoyl10"
  >
    <div>
      <div
        class="react-datepicker__input-container"
      >
        <input
          class="form-control"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
